pennylane/devices/autograd_ops.py

```diff
@@ -180,3 +180,104 @@ def MultiRZ(theta, n):
         array[complex]: diagonal part of the multi-qubit rotation matrix
     """
     return np.exp(-1j * theta / 2 * pauli_eigs(n))
+
+
+def DoubleExcitation(phi):
+    r"""
+    Double excitation rotation.

```

Comment:
recurring suggestion
```suggestion
    r"""Double excitation rotation.

```